packages/aft/lib/src/models.dart

```diff
@@ -49,26 +68,67 @@ class PackageInfo
   const PackageInfo({
     required this.name,
     required this.path,
-    required this.usesMonoRepo,
     required this.pubspecInfo,
     required this.flavor,
   });
 
+  /// Returns the [PackageInfo] found in [dir].
+  static PackageInfo? fromDirectory(Directory dir) {
+    final pubspecInfo = dir.pubspec;
+    if (pubspecInfo == null) {
+      return null;
+    }
+    final pubspec = pubspecInfo.pubspec;
+    return PackageInfo(
+      name: pubspec.name,
+      path: dir.path,
+      pubspecInfo: pubspecInfo,
+      flavor: pubspec.flavor,
+    );
+  }
+
   /// The name of the package.
   final String name;
 
   /// Absolute path to the package.
   final String path;
 
-  /// Whether the package uses the mono_repo tool.
-  final bool usesMonoRepo;
-
   /// The package's pubspec.
   final PubspecInfo pubspecInfo;
 
   /// The package flavor, e.g. Dart or Flutter.
   final PackageFlavor flavor;
 
+  /// The integration test directory within the enclosing directory, if any
+  Directory? get integTestDirectory {
```

Comment:
This isn't used currently, correct? Are you workflows for integ tests could be generated as well?

Reply:
It's not currently used. Needed for `aft test` though. I can remove it here.